api-gateway/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
```

Comment:
What is the purpose of this api-gateway directory?

Reply:
In addition to the GCS transport contribution we also thought to provide a basic Api gateway. This service accepts POST calls
/transfer and /status. 
They can be used to trigger the gRPC calls in mft Apiserviceapplication to initiate transfer between provides. 
The /transfer POST call accepts sourceId,destId etc as body params to trigger the transfer.
We hoped this would help enable MFT to be treated standalone.

Reply:
There an standard way to convert grpc to REST using grpc gateway https://github.com/grpc-ecosystem/grpc-gateway. However it is always good to look for other alternatives but it should be a separate PR. This PR is focused on GCP transport for MFT